alphazero/_game.py

```diff
@@ -383,3 +384,339 @@ def n_players():
         Get the number of players in the game.
         """
         return 2
+
+
+class Connect4(Game):
+    def __init__(self, width=7, height=6, board_state=None):
+        """
+        Construct an instance of the game class.
+        """
+        if board_state is None:
+            self._board = self.get_initial_board(height, width)
+        else:
+            tensor = torch.Tensor(board_state)
+            if width * height != tensor.numel():
+                raise ValueError(
+                    f"width and height dimensions {width}, {height} do not "
+                    f"match board state size {tensor.shape}"
+                )
+            self._board = tensor.view(height, width)
+        self._move_count = 0
+        self._players = [0, 1]
+        self.win_list = [False, False]
```

Comment:
what is the purpose of this var?

Reply:
Deleted it, was a product of something I was trying with winning_move